spec/openapi.yml

```diff
@@ -963,6 +963,10 @@ components:
           type: string
         facets:
           $ref: '#/components/schemas/DatasetFacets'
+        currentVersionUuid:
```

Comment:
For versions, we don't specify that they are `UUIDs`. For example, in our [DatasetVersionAPI](https://marquezproject.github.io/marquez/openapi.html#tag/Datasets/paths/~1namespaces~1{namespace}~1datasets~1{dataset}~1versions~1{version}/get), we use `Dataset.version`. Mind using `currentVersion`?

Reply:
I can, but we have already such field in Dataset.
https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/service/models/Dataset.java#L64
Just made in the same way as it is

Reply:
Yeah, for public fields, I wouldn't specify the UUID format 👍